test/JDBC/expected/BABEL-3614-vu-verify.out

```diff
@@ -202,8 +204,9 @@ SELECT BABEL_3614_vu_prepare_f7()
 GO
 ~~START~~
 datetime2
-<NULL>
-~~END~~
```

Comment:
Just want to make sure, is this the same behavior as SQL Server - do they return NULL or throw an error in this case?

Reply:
For this case and all the other ones like it in this file*

Reply:
Yes this is how sql server handles it. If conversion to that type is prohibited it throws an error. If conversion to that type is allowed but the conversion cannot be completed it returns null. Will run through all of the test cases to confirm

Reply:
Yeah so everything behaves the same aside from some of the rounding which just has to do with our difference in representation. Errors are thrown in the same cases though